testsuite/MDAnalysisTests/transformations/test_translate.py

```diff
@@ -34,28 +34,116 @@
 def translate_universes():
     # create the Universe objects for the tests
     reference = make_Universe(trajectory=True)
-    transformed = make_Universe(trajectory=True)
-    transformed.trajectory.ts.dimensions = np.array([372.,373.,374.,90,90,90])
+    transformed = make_Universe(['masses'], trajectory=True)
+    transformed.trajectory.ts.dimensions = np.array([372., 373., 374., 90, 90, 90])
     return reference, transformed
 
-def test_translate():
-    ref_u, trans_u = translate_universes()
+def test_translate_coords(translate_universes):
+    ref_u = translate_universes[0]
+    trans_u = translate_universes[1]
     ref = ref_u.trajectory.ts
-    vector = np.float32([1,2,3])
+    vector = np.float32([1, 2, 3])
     ref.positions += vector
     trans = translate(vector)(trans_u.trajectory.ts)
     assert_array_almost_equal(trans.positions, ref.positions, decimal=6)
-    
 
-def test_center():
+def test_translate_vector(translate_universes):
```

Comment:
This makes me realize: how is the dtype of `ts.positions` affected by the dtype of the vector?

Reply:
I had an error in #1902 where if the vector was not  `np.float32` some floating point errors would pop up